Sources/DolbyIORTSUIKit/Public/Screens/Media/StreamingScreen.swift

```diff
@@ -13,10 +13,13 @@ public struct StreamingScreen: View {
     @State private var isShowingSettingsScreen: Bool = false
     @ObservedObject private var themeManager = ThemeManager.shared
     
+    let primaryVideoQuality: VideoQuality
+    
     private var theme: Theme { themeManager.theme }
     
-    public init(isShowingStreamView: Binding<Bool>) {
+    public init(isShowingStreamView: Binding<Bool>, primaryVideoQuality: VideoQuality = .auto) {
```

Comment:
There is no way to change the `primaryVideoQuality` from the UI yet ?

Reply:
No such req in the ultra app. Settings need to be updated as well. So much work for a testing app. Only plan to set it from the viewer app (host)